billy/web/api/handlers.py

```diff
@@ -490,7 +485,7 @@ def _ocd_id_to_shape_url(self, ocd_id):
         )
         data = json.load(urllib2.urlopen(url))
         geometries = filter(
-            lambda x: x['boundary_set']['name'] in ACTIVE_BOUNDARY_SETS,
+            lambda x: re.search(ACTIVE_BOUNDARY_SETS_RE, x['boundary_set']['name']),
```

Comment:
We should perhaps do a re.match  + handling for `sldl-nh-12-floterial` or something